python/ray/data/dataset.py

```diff
@@ -2188,8 +2188,8 @@ def get_feature_tensors(
     def to_tf(
```

Comment:
Can you split this into a separate PR against datasets?

Reply:
https://github.com/ray-project/ray/pull/22916